src/App/Tracker/Controller/Hooks/Listeners/AbstractListener.php

```diff
@@ -289,4 +286,107 @@ protected function createStatus(Github $gitHub, TrackerProject $project, $issueN
 			$status->state, $status->targetUrl, $status->description, $status->context
 		);
 	}
+
+	/**
+	 * Set Categories
+	 *
+	 * @param   object       $hookData       Hook data payload
+	 * @param   Logger       $logger         Logger object
+	 * @param   object       $project        Object containing project data
+	 * @param   IssuesTable  $table          Table object
+	 * @param   array        $addCategories  The Categories to add
+	 *
+	 * @return  void
+	 *
+	 * @since   1.0
+	 * @throws  \RuntimeException
+	 */
+	protected function setCategories($hookData, Logger $logger, $project, IssuesTable $table, $addCategories)
+	{
+		// The Github ID if we have a pull or issue so that method can handle both
+		$issueNumber = $this->getIssueNumber($hookData);
+
+		if ($issueNumber === null)
+		{
+			$logger->error(
+				sprintf(
+					'Error retrieving issue number for %s/%s',
+					$project->gh_user,
+					$project->gh_project
+				)
+			);
+
+			throw new \RuntimeException('Error retrieving issue number for ' . $project->gh_user . '/' . $project->gh_project);
+		}
+
+		$categoryModel            = new CategoryModel($this->getContainer()->get('db'));
+		$category['issue_id']     = $table->id;
+		$category['modified_by']  = $this->getGithubBotName($project);
+		$category['categories']   = $addCategories;
+		$category['issue_number'] = $issueNumber;
+		$category['project_id']   = $project->project_id;
+
+		$categoryModel->updateCategory($category);
```

Comment:
hmm i can't see the IDs in the jissues backed can you send me a list of category to id? (e.g. via Glip / mail?